modules/ROOT/pages/traversal-framework/bidirectional_traversal_framework.adoc

```diff
@@ -1,21 +1,19 @@
-:description: The Neo4j Bidirectional Traversal Framework Java API.
+:description: Neo4j Bidirectional Traversal Framework Java API.
 
 :org-neo4j-graphdb-bidirectional-traversal-description: {neo4j-javadocs-base-uri}/org/neo4j/graphdb/Transaction.html#bidirectionalTraversalDescription()
 
 [[Bidirectional-Traversal-Framework]]
-= The Bidirectional Traversal Framework
+= Bidirectional Traversal Framework
 
 A bidirectional traversal consists of two traversals starting from different nodes, returning the paths where both traversals collide.
+The Bidirectional Traversal Framework allows the description of such a traversal with the link:/java-reference/{neo4j-version}/traversal-framework/bidirectional_traversal_framework/#_defining_the_bidirectional_traverser[BidirectionalTraversalDescription].
 
-The Bidirectional Traversal Framework allows the description of such a traversal with the link:{org-neo4j-graphdb-bidirectional-traversal-description}[BidirectionalTraversalDescription^].
-
-Here's a visual representation of a bidirectional traversal from start node `1` and end node `5`,
-where all bold relationships are in the result paths given the following restrictions:
+image:bidirectional-example.png[role="middle"]
 
-* The start side traverser only traverses relationships of type `A`
-* The end side traverser only traverses relationships of type `B`
+Here's a visual representation of a bidirectional traversal from start node `1` and end node `5`, where all bold relationships are in the result paths given the following restrictions:
```

Comment:
Later on you changed Here's to Here is, I am not sure how formal we want the docs, but could be nice with consistency :)